pootle/apps/pootle_translationproject/views.py

```diff
@@ -233,7 +233,22 @@ class TPBrowseView(TPDirectoryMixin, TPBrowseBaseView):
 
     @cached_property
     def items(self):
-        return get_children(self.object)
+        dirs_with_vfolders = set(
```

Comment:
is it worth to create a function or method for it since you you the same code in tests?